tests/test_messages.py

```diff
@@ -0,0 +1,137 @@
+import json
+import responses
+import pytest
+
+from trycourier.client import Courier
+from trycourier.exceptions import CorierAPIException
+
+@responses.activate
+def test_success_messages_list():
+    responses.add(
+        responses.GET,
+        'https://api.courier.com/messages',
+        status=200,
+        content_type='application/json',
+        body='{"paging": {}, "items": []}'
+    )
+
+    c = Courier(auth_token='123456789ABCDF')
+    r = c.messages.list()
+
+    assert r == {'paging': {}, 'items': []}
+
+
+@responses.activate
+def test_success_messages_list_with_params():
+    responses.add(
+        responses.GET,
+        'https://api.courier.com/messages?cursor=1234&event=my.event.id&recipient=1234',
```

Comment:
This line is 88 characters long which upsets flake8. Need to figure out a way make this line smaller.